index.Rmd

```diff
@@ -1,105 +1,299 @@
+band6 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b06.tiff")
+band7 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b07.tiff")
+band8 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b08.tiff")
+band9 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b09.tiff")
+```
 
+Show result of band1
 
-Here's my first code chunk.
 ```{r}
-1+2
+print(band1)
 ```
 
-Load any required packages in a code chunk (you may need to install some packages):
+Now, I check the size of image before combining them together using STACK to make sure the sizes match. The code below shows that band1, band2, band3, and band7 are 10x10. Band4, band5, and band6 are 20x20.
 
-```{r, message=F, warning=F}
-library(tidyverse)
-library(leaflet)
-library(kableExtra)
-knitr::opts_chunk$set(cache=TRUE)  # cache the results for quick compiling
+
+```{r}
+#Here is an example for Landsat data of different size
+res(Band1) # Size find is 10x10
+res(Band2) # Size find is 10x10
+res(Band3) # Size find is 10x10
+res(Band4) # Size find is 20x20
+res(Band5) # Size find is 20x20
+res(Band6) # Size find is 20x20
+res(Band7) # Size find is 10x10
+
+#Here is the code use to find size of Martian data
+res(band1) # Size find is 10x10
+res(band2) # Size find is 10x10
+res(band3) # Size find is 10x10
+res(band4) # Size find is 20x20
+res(band5) # Size find is 20x20
+res(band6) # Size find is 20x20
+res(band7) # Size find is 10x10
+res(band8) # Size find is 10x10
+res(band9) # Size find is 10x10
 ```
 
-## Download and clean all required data
+I resize now some image to make sure all image have the same size before adding them together.
+From the results above, I then multiply each image (band1, band2, band3, and band7) by 2 to have 20x20 for Earth data. However, for Martian data, we have the same size so we will just add them all together later without resizing any image.
+
+
 ```{r}
-n=20
-data=data.frame(x=runif(n,-180,180),
-                y=runif(n,-60,60),
-                size = runif(n, 5, 20),
-                category = factor(
-                  sample(letters[1:5], n, replace = TRUE)
-                  ),
-                value = rnorm(n))
+#This is for Earth (Landsat) data to show how it works
+band1 <- aggregate(band1, fact = 2) # multiply the size of band1 by 2 to have 20x20
```

Comment:
aggregate is doing the opposite - it's combining 2x2 pixels into  1x1.  Check the help file for this